research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -288,13 +337,13 @@ The term "light dismiss" for a popup is used to describe the user "moving on" to
 
 ### Nested Popups
 
-For at least `popup=popup`, it is possible to have "nested" popups. I.e. two popups that are allowed to both be open at the same time, due to their relationship with each other. A simple example where this would be desired is a popup menu that contains sub-menus: it should be possible to keep the main menu showing while the sub-menu is shown.
+For at least `popup=auto`, it is possible to have "nested" popups. I.e. two popups that are allowed to both be open at the same time, due to their relationship with each other. A simple example where this would be desired is a popup menu that contains sub-menus: it should be possible to keep the main menu showing while the sub-menu is shown.
```

Comment:
```suggestion
For at least `popup=auto`, it is possible to have "nested" popups. I.e. two popups that are allowed to both be open at the same time, due to their relationship with each other. A simple example where this would be desired is a popup menu that contains sub-menus: it is commonly expected to support this pattern, and keep the main menu showing while the sub-menu is shown.
```

Reply:
Done.